app/assets/stylesheets/application.css.scss

```diff
@@ -518,4 +518,19 @@ span.twitter-typeahead {
 
 .paper-preview-not-available {
   text-align: center;
+}
+
+.latest-run {
+  font-size: 0;
+  line-height: 0;
+  height: 14px;
+  width: 14px;
+  display: inline-block;
+  vertical-align: middle;
+  border: 7px solid #9b0000;
```

Comment:
Color literals like `#9b0000` should only be used in variable declarations; they should be referred to via variable everywhere else.